kitsune/groups/forms.py

```diff
@@ -1,6 +1,6 @@
 from django import forms
 from django.conf import settings
-from django.utils.translation import ugettext as _
+from django.utils.translation import gettext as _
 from django.utils.translation import gettext_lazy as _lazy
```

Comment:
do we still need this?

Reply:
Looks like it - used in line 28:

`        self.fields["avatar"].help_text = _("Your avatar will be resized to {size}x{size}").format(
            size=settings.AVATAR_SIZE
        )`